### PR TITLE
fix: lint changes for eslint 4.x

### PR DESCRIPTION
**Pull request by petey**
Context
=======
ESLint 4.x included a rewrite of the `indent` rule that is much more strict. http://eslint.org/docs/user-guide/migrating-to-4.0.0#indent-rewrite

Objective:
----------
Fix all lint errors due to indent rule changes introduced by eslint 4.x

Misc:
------
`eslint --fix .` is nice.
Apologies ahead of time to Yahoo! Japan, as some of my previous indentation guidance is not consistent with this change.

### CODE REVIEW

File: plugins/builds/create.js [hunk 1]
```diff
@@ -39,59 +39,59 @@ module.exports = () => ({
...
             // scmUri is buried in the pipeline, so we get that from the job
-            .then(([job, user]) => job.pipeline.then(pipeline =>
-                user.getPermissions(pipeline.scmUri)
-                // check if user has push access
-                .then((permissions) => {
-                    if (!permissions.push) {
-                        throw boom.unauthorized(`User ${username} `
+                .then(([job, user]) => job.pipeline.then(pipeline =>
+                    user.getPermissions(pipeline.scmUri)
+                    // check if user has push access
```
Comment (joelseq) [inline plugins/builds/create.js]: should this be the same level of indentation as the `.then` below it?

File: plugins/builds/create.js [hunk 2]
```diff
@@ -39,59 +39,59 @@ module.exports = () => ({
...
-                    token,
-                    scmUri: pipeline.scmUri,
-                    prNum: job.prNum
-                }))
-                .then((sha) => {
-                    let type = 'pipeline';
-                    let workflow = pipeline.workflow;
+                            }
+                        })
+                    // user has good permissions, sync and create a build
```
Comment (joelseq) [inline plugins/builds/create.js]: should this comment be the same level of indentation as the `.then` below it?

File: plugins/builds/create.js [hunk 3]
```diff
@@ -39,59 +39,59 @@ module.exports = () => ({
...
+                        .then((event) => {
+                            payload.eventId = event.id;
 
-                    return buildFactory.create(payload);
-                })
-            )).then((build) => {
+                            return buildFactory.create(payload);
+                        })
+                )).then((build) => {
                 // everything succeeded, inform the user
```
Comment (joelseq) [inline plugins/builds/create.js]: same question regarding level of indentation for comment as ones above

Comment (ian-fox) [inline plugins/builds/create.js]: This is code that has been removed.

File: plugins/builds/create.js (hunk 1 above)
Comment (petey) [inline plugins/builds/create.js]: Thanks, fixed.

**Pull request merged**